packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/contact/component.ts

```diff
@@ -55,6 +61,11 @@ export default class SidebarContact extends Component<IArgs> {
       return true;
     }
 
+    // always show if contact is pinned
```

Comment:
codeClimate complains about having too many return statements in this class. Do you want me to mix all the ones with 'return true'  into one if statement?

Reply:
naw, clarity of intent always wins of conciseness

Reply:
however, there may be a way to convert this and still have clarity of intent, maybe...
```ts
let shouldRender = (
  contact.id === currentUserId
  // always show if online
  || contact.onlineStatus !== STATUS.OFFLINE
  // always show if there are unread messages
  || this.hasUnread
  // always show if contact is pinned
  || this.isPinned

);

```